src/github.com/matrix-org/go-neb/services/circleci/circleci.go

```diff
@@ -0,0 +1,276 @@
+// Package circleci implements a Service capable of processing webhooks from CircleCI.
+package circleci
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"regexp"
+	"strings"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/matrix-org/go-neb/database"
+	"github.com/matrix-org/go-neb/types"
+	"github.com/matrix-org/gomatrix"
+	"strconv"
+)
+
+// ServiceType of the CircleCI service.
+const ServiceType = "circleci"
```

Comment:
webhook-circleci

Reply:
github has the webhook as suffix. I will keep the Github syntax

Reply:
and `WebhookServiceType` for consistency :P

Reply:
Also travisCI isn't namespaced as a webhook

Reply:
Ping @t3chguy ^

Reply:
yeah ignore this because of github-webhooks being a specialcase